sdk/src/main/java/co/omise/android/models/SourceType.kt

```diff
@@ -34,6 +34,11 @@ sealed class SourceType(
         data class Unknown(@JsonValue override val name: String?) : InternetBanking(name)
     }
 
+    sealed class MobileBanking(@JsonValue override val name: String?) : SourceType(name) {
+        object Scb : MobileBanking("mobile_banking_scb")
```

Comment:
Not sure future plan, will we have more new MobileBankings in the future?

Reply:
Yes, probably

Reply:
according to PO we will have BAY KBANK and more in future khub